baseplate/experiments/providers/feature_flag.py

```diff
@@ -7,7 +7,10 @@
 
 
 class FeatureFlag(R2Experiment):
-    """An experiment with a single variant "active".
+    """.. deprecated:: 0.27
```

Comment:
the deprecated thing should go near the bottom of the docstring, not replace the summary line. you can run `make docs` in the vm and navigate to `http://baseplate.local/html` to see what the output of sphinx looks like to make sure it all makes sense.

Reply:
Would it make sense to have it at the top of the docstring, but not replace the summary line? Seems to me the _first_ thing you'd want to know when reading a block of docs is that the class is deprecated?